lib/utils/connect.js

```diff
@@ -0,0 +1,21 @@
+const { parse } = require('url');
+const mongoose = require('mongoose');
+
+function redact(uri) {
+  const parsedUri = parse(uri);
+  const authPart = parsedUri.auth ? '****:****@' : '';
+  return `${parsedUri.protocol}://${authPart}${parsedUri.hostname}:${parsedUri.port}${parsedUri.path}`; 
+}
+
+function log(event, dbUri) {
+  return function() {
+    console.log(`Connection ${event} on ${redact(dbUri)}`);
+  };
+}
+
+module.exports = (dbUri = process.env.MONGODB_URI) => {
+  mongoose.connect(dbUri, { useNewUrlParser: true });
```

Comment:
```suggestion
  mongoose.connect(dbUri, { useNewUrlParser: true, useCreateIndex: true, });
```
This will resolve the deprecation warning that is currently popping up.